host/tests/rmfAudio_L3_TestCases/rmfAudio_test01_primaryDataCapture.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+import time
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../"))
+
+from rmfAudio_L3_TestCases.rmfAudioHelperClass import rmfAudioHelperClass
+from raft.framework.core.logModule import logModule
+
+class rmfAudio_test01_primaryDataCapture(rmfAudioHelperClass):
+    """
+    Test class to open capture Primary Audio Data and verify it with its source.
+
+    This class uses the `rmfAudioClass` to test primary data capture,
+    downloading necessary test assets, setting up RMF Audio Capture
+    and performing verification of captured audio.
+    """
+    def __init__(self, log:logModule=None):
+        """
+        Initializes the test class with test name, setup configuration, and sessions for the device.
+
+        Args:
+            None
+        """
+        self.testName  = "test01_primaryDataCapture"
+        super().__init__(self.testName, '1', log)
+
+    def testAudioCapture(self, file_path:str):
+        """
+        Verifies if the captured audio data is a match with input reference
+
+        Args:
+            file_path (str): File name and path of the output wav file created
+
+        Returns:
+            bool: True if audio capture is a match, False otherwise.
+        """
+        test = self.testSetup.get("assets").get("device").get(self.testName)
+
+        # Download test streams to device
+        url =  test.get("streams")
+        if url is not None:
+            ref_file_path = self.testStreams[0]
+            ref_file_path = ref_file_path.replace("\\", "/")

```

Comment:
here also not required 
ref_file_path = ref_file_path.replace("\\", "/")

Reply:
updated as suggested